code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -50,7 +48,15 @@
 			/mob/living/simple_animal/hostile/blob,
 			/mob/living/simple_animal/ascendant_shadowling
 			)//exclusion list for things you don't want the reaction to create.
-		var/list/critters = typesof(/mob/living/simple_animal/hostile) - blocked // list of possible hostile mobs
+		var/list/meancritters = typesof(/mob/living/simple_animal/hostile) - blocked // list of possible hostile mobs
+		var/list/nicecritters = list(typesof(/mob/living/simple_animal/pet),

```

Comment:
it's valid syntax for list in list, which is not how these critter lists work. ```pick()``` is not smart enough to choose an element from the embedded list, it just returns the list.

tl;dr this is wrong, it needs to be 
```nicecritters += typesof(/mob/living/simple_animal/pet) - /mob/living/simple_animal/pet```